lib/main.dart

```diff
@@ -53,33 +56,75 @@ class RouteDetailsWidget extends StatelessWidget {
     return ReorderableListView(
       onReorder: route.move,
       children: [
-        for (var c in route.places)
-          RouteCellWidget(c),
+        for (var c in route.places) RouteCellWidget(c),
       ],
     );
   }
 }
 
+class MyScaffold extends StatefulWidget{
+  @override
+  _MyScaffoldState createState() => _MyScaffoldState();
+}
+
+class _MyScaffoldState extends State<MyScaffold>{
+  bool expanded = true;
+  var rightPanelWidth;
+  var buttonWidth;
+  @override
+  Widget build(BuildContext context) {
+    final mediaQuery = MediaQuery.of(context);
+    final width = mediaQuery.size.width;
+    rightPanelWidth = max(width / 4, 228.0);
```

Comment:
Why `228`?

Reply:
I conducted an expensive high-precision study and it's the optimal size for this panel.